packages/react/src/components/TextArea/TextArea.js

```diff
@@ -142,7 +224,9 @@ TextArea.propTypes = {
    * Optionally provide an `onChange` handler that is called whenever <textarea>
    * is updated
    */
-  onChange: PropTypes.func,
+  onChange: !useControlledStateWithValue
```

Comment:
Oops forgot to mention that, my apologies - For `<TextInput>` and `<TextArea>` we don't have a breaking change, and thus the feature flag won't be needed.

Reply:
oh yeah we discussed this previously, reverted the commit

Reply:
@emyarod I meant `onChange: requiredIfValueExists(PropTypes.func)` - Wondering if that's what you did.

Reply:
yup updated @asudoh

Reply:
Cool thanks @emyarod!